practice-one/src/styles/components/footer.css

```diff
@@ -0,0 +1,91 @@
+.footer-container {
+  margin-top: 158px;
+}
+
+.website-summary-container {
+  display: flex;
+  justify-content: space-between;
+}
+
+.brief-description {
+  width: 325px;
+  padding-left: 56px;
+  margin-right: 109px;
+}
+
+.brief-heading {
+  color: var(--color-primary,#F54748);
+  font-weight: var(--font-weight-sub-heading, 600);
+  line-height: var(--line-height-normal, 1.3);
+  font-size: var(--font-size-extra-large, 32px);
+}
+
+.brief-content {
+  margin-top: 10px;
+  margin-bottom: 23px;
+  font-size: var(--font-size-normal, 18px);
+}
+
+.brief-icon-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100px;
+}
+
+.company-information-container {
+  margin: 0 94px 0 99px;
+}
+
+.about-us-heading,
+.company-information-heading {
+  color: var(--color-primary, #F54748);
+  font-size: var(--font-size-large, 48px);
+  font-weight: var(--font-weight-sub-heading, 600);
+}
+
+.about-us-item,
+.company-information-item {
+  margin-top: 14px;
+}
+
+.about-us-item > a,
+.company-information-item > a {
+  font-size: var(--font-size-normal, 18px);
+  color: var(--color-dark, #191919);
+  text-decoration: none;
+}
+
+.subscribe-form {
+  width: 379px;
+}
+
+.subscribe-heading {
+  font-size: var(--font-size-large, 48px);
+  color: var(--color-primary, #F54748);
+  font-weight: var(--font-weight-heading, 700);
+}
+
+.subscribe-content {
+  line-height: var(--line-height-normal, 1.3);
+  margin: 37px 0 27px 0;
+  font-size: var(--font-size-normal, 18px);
+}
+
+.subscribe-input > input[type="text"] {
+  width: 205px;
+  height: 50px;
+  border-radius: 38px;
+  padding-left: 20px;
+  background: var(--color-neutral, #C4C4C4);
```

Comment:
recheck this value, do we have variable for this

Reply:
I added a variable for this